kits/debate-arena/prompts/debate-judge_instructorllmnode-239_user_1.md

```
@@ -0,0 +1 @@
+Topic: {{triggerNode_1.output.topic}} Position A: {{triggerNode_1.output.positionA}} Position B: {{triggerNode_1.output.positionB}} Debate Transcript: {{triggerNode_1.output.transcript}} Evaluate the debate above and produce your verdict as valid JSON matching the output schema.
```

markdownlint-cli2 (0.22.1)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)